FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/viewmodel/EkmViewModel.kt

```diff
@@ -52,15 +56,29 @@ class EkmViewModel(application: Application) : BaseAndroidViewModel(application)
         val ekmPrivateResult = repository.getPrivateKeysViaEkm(
           context = context,
           ekmUrl = orgRules.keyManagerUrl
-            ?: throw java.lang.IllegalArgumentException("key_manager_url is empty"),
+            ?: throw IllegalArgumentException("key_manager_url is empty"),
           tokenId = tokenId
         )
 
         if (ekmPrivateResult.data?.privateKeys?.isEmpty() == true) {
-          throw java.lang.IllegalStateException(context.getString(R.string.no_prv_keys_ask_admin))
+          throw IllegalStateException(context.getString(R.string.no_prv_keys_ask_admin))
```

Comment:
This is good if it's handled in a way that doesn't get reported to ACRA. It may be pretty common that user signs up and doesn't have any keys yet.

Reply:
I don't think that we should send any ACRA reports from here. Here can be sensitive info. I handle all exceptions and show them to a user. Please let me know if I'm wrong

Reply:
Ok. Sounds good